steamworks_unix.go

```diff
@@ -53,6 +53,10 @@ import (
 //   return ((bool (*)(void*, void*, void*, int32_t))(f))((void*)arg0, (void*)arg1, (void*)arg2, arg3);
 // }
 //
+// static uint8_t callFunc_Bool_Ptr_Int32_Int32_Int32_Int32_Int32(uintptr_t f, uintptr_t arg0, int32_t arg1, int32_t arg2, int32_t arg3, int32_t arg4, int32_t arg5) {
```

Comment:
Keep the alphabetical order (i.e. this should come after callFunc_Bool_Ptr_Bool)